syntax = "proto3";

package proto;

service Embed {
  rpc Embed (EmbedRequest) returns (EmbedResponse);
}

message EmbedRequest {
  string text = 1;
  bool is_query = 2;
}

message EmbedResponse {
  repeated float embedding = 1;
}